webpack.config.js

```diff
@@ -12,6 +12,7 @@ const projectRoot = 'cms';
 const options = {
   entry: {
     main: `./${projectRoot}/static_src/javascript/main.js`,
+    auth: `./${projectRoot}/auth/static/js/auth.js`,
```

Comment:
Issue: I think these should stay in the `static_src` dir, so the scripts are kept together, and linters definitely run.

Reply:
Done!